spec/services/manual/create_service_spec.rb

```diff
@@ -34,6 +34,25 @@
     end
   end
 
+  context 'when the manual is valid but saving it to the publishing api fails' do
```

Comment:
Minor: I think we've been capitalising "Publishing API" in most places. It would be nice to be consistent.